tests/Command/pdg.config.yaml

```
@@ -1,14 +1,11 @@
 pdg:
   references:
-    src: 'tests/Command/src'
-    namespace: 'PhpDocumentGenerator\Tests\Command\App'
-    patterns:
-      directories: [ '' ]
-      names: [ '*.php' ]
-      exclude: [ '*Interface.php' ]
-      class_tags_to_ignore: [ '@internal', '@experimental' ]
-    output: 'tests/Command/pages/references'
     base_url: '/pages/references'
+    class_tags_to_ignore: [ '@internal', '@experimental' ]
+    exclude: [ '*Interface.php' ]
```

```suggestion
    exclude: [ 'tests/Command/src/Services/*Interface.php' ]
```
This will ensure fnmatch usage

---

Will be fixed in #36